server/integration/testsuite/src/test/java/org/infinispan/server/test/configs/ExampleConfigsIT.java

```diff
@@ -653,13 +668,17 @@ private void doPutGetWithExpiration(RemoteInfinispanMBeans s1, RemoteInfinispanM
 
     private void doPutGet(RemoteInfinispanMBeans s, RemoteCache<Object, Object> c) {
         assertEquals(0, s.cache.getNumberOfEntries());
-        for (int i = 0; i < 10; i++) {
-            c.put("k" + i, "v" + i);
-        }
-        for (int i = 0; i < 10; i++) {
-            assertEquals("v" + i, c.get("k" + i));
-        }
-        assertEquals(10, s.cache.getNumberOfEntries());
+
+        final int maxRange = 10;
+        Map<String, String> testMap = new HashMap<>();
+        IntStream.range(0, maxRange)
+                .forEach(number -> testMap.put("k" + number, "v" + number));
+
+        c.putAll(testMap);
```

Comment:
Considering that the test also verifies the cache contents with `retrieveEntries()` instead of `get()`, I'd argue that this isn't fixing the existing test, it's just testing something different.